lldb/source/Plugins/Process/MacOSX-Kernel/ProcessKDP.cpp

```diff
@@ -102,10 +102,10 @@ void ProcessKDP::Terminate() {
 
 lldb::ProcessSP ProcessKDP::CreateInstance(TargetSP target_sp,
                                            ListenerSP listener_sp,
-                                           const FileSpec *crash_file_path,
+                                           FileSP crash_file_sp,
                                            bool can_connect) {
   lldb::ProcessSP process_sp;
-  if (crash_file_path == NULL)
+  if (crash_file_sp == NULL)
```

Comment:
Since you're already here, you can drop the `== NULL` and make it a little nicer.